test/stream.test.js

```diff
@@ -168,3 +168,54 @@ test('Destroying streams prematurely', t => {
     })
   })
 })
+
+test('Destroying streams prematurely with custom error handler set', t => {
+  t.plan(5)
+
+  const fastify = Fastify()
+  const stream = require('stream')
+  const http = require('http')
+
+  fastify.setErrorHandler(function (err, reply) {
+    t.ok(err)
+    try {
+      reply.send('error message')
```

Comment:
this should be an `Error`  object.

Reply:
I added another try-catch that tries to send an `Error` object. Although it doesn't really matter what gets sent because they both end up here with `this.sent` being `true`:
https://github.com/fastify/fastify/blob/a94ffdc2be92300f2bfa566a80c8dc6cdc39c6de/lib/reply.js#L31-L35

Reply:
Oh yes, my bad.